templates/SilverStripe/Admin/LeftAndMain_Menu.ss

```diff
@@ -5,7 +5,8 @@
         <% include SilverStripe\\Admin\\LeftAndMain_MenuLogo %>
         <% include SilverStripe\\Admin\\LeftAndMain_MenuStatus %>
 
-        <% if $ListSubsites.Count > 1 %>
+        <% if $SubsiteSwitchList.Count > 1 %>
+            <% require javascript('silverstripe/subsites:client/dist/js/LeftAndMain_Subsites.js') %>
```

Comment:
Moved the require code here instead of in the method since we don't need to re-require it for the xhr requests and because frankly a require call in an arbitrary PHP method is just gross